.github/workflows/tests_and_qc.yml

```diff
@@ -0,0 +1,57 @@
+# Built from:
```

Comment:
@Sigfried I decided to copy/paste the GH action that OAK uses and repurpose that. If you want to see what that action looked like in full, you can check out the first commit of this PR. It had some additional steps that one day in the future we could possibly include, e.g. _lint_ and _coverage_.